app/controllers/work_admins_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# a controller for the admins for a work.
+class WorkAdminsController < ApplicationController
```

Comment:
Can you follow the pattern established in #2454 and make this a LocksController or similar?

Reply:
Ok, did it this way under the assumption that we may want to use this controller for more than just locking (assuming there are other work specific admin functions), but we could also just create more controllers in that case.